src/templates/ar/2019/base.html

```diff
@@ -65,6 +65,7 @@
         {{ language_switcher() }}
       </nav>
       <p class="copyright"><span>© 2019 Web Almanac. Licensed under <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/LICENSE">Apache 2.0</a>.</span></p>
+      <p class="accessibility"><a href="{{ url_for('accessibility_statement', year=year, lang=lang) }}">Accessibility</a></p>
```

Comment:
It's not clear what this link means. We have an accessibility chapter and this might cause some confusion with that. I proposed a change above to merge it with the copyright notice paragraph. That may also help simplify the CSS changes.

Reply:
I originally had "Accessibility Statement" and this is what the W3C has.

However just "Accessibility" seems much more common as I discovered when I looked around to find the French and Spanish translation of this. For example [Apple](https://www.apple.com/), [Microsoft](https://www.microsoft.com/en-us/) and [your own, little known, company](https://about.google/?) 😀

Reply:
Sure, but in the context of the Almanac, "Accessibility" is also the title of a chapter so I think it's ambiguous.

Reply:
Fair point!

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/665/commits/1638e53c39d5a178076de0e81a8e981a6f29bd9e